### PR TITLE
Fix problem with legacy ms-dos paths (8.3 format)

### PR DESCRIPTION
**Pull request by pblaz1988**
We have experienced problems with opening files with legacy ms-dos paths (8.3 format), from example from Lotus Notes.

Example: Imageglass was not able to open file "C:\Users\BLAZ~1.POT\AppData\Local\Temp\notesXXXX\slika test.JPG" which didn't translate to "C:\Users\blaz.potocnik\AppData\Local\Temp\notesXXXX\slika test.JPG"

### CODE REVIEW

File: Source/ImageGlass/frmMain.cs [hunk 1]
```diff
@@ -350,8 +350,17 @@ private void LoadImages(List<string> imageFilenameList, string filePath)
             // Find the index of current image
             if (filePath.Length > 0)
             {
+
+                // this part of code fixes calls on legacy 8.3 filenames
+                // (for example opening files from IBM Notes)
+                var di = new DirectoryInfo(filePath);
+                filePath = di.FullName;
+
+                var filePathFixed = App.ToAbsolutePath(filePath);
```
Comment (d2phap) [inline Source/ImageGlass/frmMain.cs]: `filePathFixed`  is not used?

**Comment by d2phap** [comment]
Thanks @pblaz1988 !
I just rebased to `develop` branch because `master` branch is for stable release.

I saw you have `filePathFixed` variable but it's not being used in the code, did you forget?

File: Source/ImageGlass/frmMain.cs (hunk 1 above)
Comment (pblaz1988) [reply Source/ImageGlass/frmMain.cs]: hi,

sorry, i forgot about this line of code. filePathFixed should not be there.

Comment (d2phap) [reply Source/ImageGlass/frmMain.cs]: anyway, thanks!

**Pull request merged**